playground/mocks/data/idp/idx/safe-mode-credential-enrollment-intent.json

```diff
@@ -0,0 +1,22 @@
+{
+  "stateHandle": "02lXCU8zPR9jbdnipnWiPcyzdwQELBBW9K2oRm1mmt",
+  "version": "1.0.0",
+  "expiresAt": "2020-11-23T01:34:19.000Z",
+  "intent": "CREDENTIAL_ENROLLMENT",
+  "remediation": {
+    "type": "array",
+    "value": []
+  },
+  "messages": {
+    "type": "array",
+    "value": [
+      {
+        "message": "Set up is temporarily unavailable due to server maintenance. Click 'Back to Settings' to continue.",
```

Comment:
I want to discuss this a little more. "Back to settings" is only available if the SIW is okta-hosted and they are in our settings page. If at some point a custom hosted widget is boostraped with intent=CREDENTIAL_ENROLLMENT, this error message won't make sense, since "Back to settings" link lives on our page that hosts the widget, not within the widget itself. 

I think the error should be something like "Try again later" only
@anishashetty-okta let me know offline where is the backend PR, I can make comments there